pxtrunner/runner.ts

```diff
@@ -468,7 +469,10 @@ namespace pxt.runner {
 
     export async function fetchSimJsInfo(simOptions: SimulateOptions): Promise<pxtc.BuiltSimJsInfo> {
         try {
-            return await pxt.Cloud.downloadBuiltSimJsInfoAsync(simOptions.id);
+            const start = Date.now();
+            const result = await pxt.Cloud.downloadBuiltSimJsInfoAsync(simOptions.id);
+            pxt.tickEvent("runner.fetchSimJsInfo", { durationMs: Date.now() - start });
```

Comment:
Food for thought: do we want 2 different events here, or is it more helpful/convenient to have one "perfMeasurement" event with a property for the event? In some cases it's helpful to have the latter, so you can easily query all of them and sort by which events are taking the longest. Trickier to do that if they all have different names.

Reply:
Fetching the already built JS info and building the JS info from the script text (which we request from the backend as well) are mutually exclusive. Though I agree that using something like `perfMeasurement` vs `runner` will make it easier to keep track of these. When you say property, you mean adding a customDimension with operation name? Just like we do with `duration` now?

Reply:
Ended up implementing this!